### PR TITLE
Feat/adapt last waterings

### PR DESCRIPTION
**Pull request by aeschi**
task: [https://app.asana.com/0/1204152181784036/1207019604438027](https://app.asana.com/0/1204152181784036/1207019604438027)

### CODE REVIEW

File: src/components/tree-detail/last-waterings/last-waterings.tsx [hunk 1]
```diff
@@ -45,9 +41,6 @@ export const LastWaterings: React.FC<LastWateringsProps> = ({
 		});
 	}, [treeWateringData]);
```
Comment (raphael-arce) [inline src/components/tree-detail/last-waterings/last-waterings.tsx:42]: I think this needs to be adjusted accordingly (maybe we need to first discuss it with our POs):
currently, you can only see waterings of the same calendar year. However, with the new 30 days prior logic, you will be able to see 30 days prior, independently of whether it is the same year or not.

Example: if you water the 01.12.2024 and 28.12.2024 then 02.01.2025 (unlikely, i know), you'll see the watering from the 28.12.2024, but not the one from the 01.12.2024.

Maybe we should not look at the calendar year, but rather the last 365 days. 

(or we just ignore this edge case)

**Review by raphael-arce — CHANGES_REQUESTED** [review]
I have a few suggestions and noticed an edge-case.

File: src/components/tree-detail/last-waterings/last-waterings.tsx (hunk 1 above)
Comment (aeschi) [reply src/components/tree-detail/last-waterings/last-waterings.tsx:42]: I talked to Henriette about it and it should actually be "all previous waterings" disregarding the year. So I will adapt it.

**Review by raphael-arce — APPROVED** [review]
lgtm 👍 🚀

**Pull request merged**